desktop-app/app/components/Addressinput/index.js

```diff
@@ -294,7 +294,7 @@ class AddressBar extends React.Component<Props> {
   _normalize = (address: string) => {
     if (address.indexOf('://') === -1) {
       let protocol = 'https://';
-      if (address.startsWith('localhost') || address.startsWith('127.0.0.1')) {
+      if (address.startsWith('localhost') || address.startsWith('127.0.0.1') || address.split(':')[0].split('?')[0].endsWith('.localhost')) {
```

Comment:
@jjavierdguezas  Will this match if the entered URL is something like: `http://test.localhost/`?

Reply:
mmmm missed that! 😅

Reply:
done